### ISSUE TITLE
Geolocation doesn't work

### ISSUE THREAD

**Issue by mlennie** [TrustWallet/dapps-browser#14]
Is geolocation implemented in the browser? Currently `navigator.permissions.query` throws an error. Does it currently work and if not how long until that is implemented?

**Comment by vikmeup** [comment]
Can you show more details on this? Android or iOS?

**Comment by mlennie** [comment]
@vikmeup Yes Android. Also when I try `navigator.geolocation.getCurrentPosition` it returns `Position Error` with `code: 1` and `message: "User denied Geolocation"`. The problem is it never asked me to allow geolocation permission

**Comment by mlennie** [comment]
https://stackoverflow.com/questions/33070698/google-maps-v3-in-android-webview-gives-error-user-denied-geolocation
https://stackoverflow.com/questions/5329662/android-webview-geolocation/5423026

**Comment by vikmeup** [comment]
@mlennie work in progress on Android. 

I will add support on iOS side in the next few days, I just need to add permissions.

**Comment by shippomx** [comment]
https://dapps.trustwalletapp.com/ is shown in a blank page.

**Comment by vikmeup** [comment]
@shippomx where do you open it?

**Comment by kolya182** [comment]
@vikmeup is this still an issue ?

**Comment by vikmeup** [comment]
I think so

**Comment by jhoughjr** [comment]
I'm having this issue in a WKWebView on macOS now. It might be related.

**Comment by armishsonkar** [comment]
Is this resolved ? Our users on Android devices on chrome have started getting the same error: code: 1 and message: "User denied Geolocation"  while calling "navigator.geolocation.getCurrentPosition". The error is observed even when the user has allowed location access to our website https://www.meanstreets.in/ 
Though it works perfectly if users access from  Desktop/Laptop
